e2e/editor.spec.ts

```diff
@@ -28,3 +28,143 @@ test.describe('Editor Component', () => {
     await expect(text).toBeVisible();
   });
 });
+
+test.describe('Editor theme if the system theme is dark', () => {
+  test.use({ colorScheme: 'dark' });
+
+  test.describe('If the user is signed in', () => {
+    test.use({ storageState: 'playwright/.auth/certified-user.json' });
```

Comment:
I was confused by this too. It turns out we _do_ setup `storageState`, but we we don't use it by default. So, any tests that require a signed-in user have to manually specify `test.use`, like you did here.

Do you think it would be more intuitive to default to using it? I'm leaning towards "yes".